### PR TITLE
AuthenticationController, MovieController and MovieSessionController added

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

File: src/main/java/com/dev/cinema/controller/AuthenticationController.java [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+public class AuthenticationController {
+    private final AuthenticationService service;
+    private final UserMapper userMapper;
+
+    @Autowired
```
Comment (RomanHreits) [inline src/main/java/com/dev/cinema/controller/AuthenticationController.java:14]: if you have one constructor, you don't seem to need an Autowired annotation

**Review by RomanHreits — APPROVED** [review]
Nice work!

File: src/main/java/com/dev/cinema/service/mapper/UserMapper.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+package com.dev.cinema.service.mapper;
+
+import com.dev.cinema.model.User;
+import com.dev.cinema.model.dto.UserRequestDto;
+import org.springframework.stereotype.Component;
+
+@Component
+public class UserMapper {
```
Comment (RomanDubovskyi) [inline src/main/java/com/dev/cinema/service/mapper/UserMapper.java]: we don't need UserMapper for now, let's remove it and add when we need it

**Review by RomanDubovskyi — COMMENTED** [review]
good job, just remove UserMapper for now

File: src/main/java/com/dev/cinema/service/mapper/UserMapper.java (hunk 2 above)
Comment (Rommelua) [reply src/main/java/com/dev/cinema/service/mapper/UserMapper.java]: done

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**